common/journal_entries/dvg_east_indies_company.txt

```diff
@@ -0,0 +1,71 @@
+je_consolidate_company_rule_in_sumatra = {
```

Comment:
Are these journals complete and working? They are never activated and are missing translations.